### PR TITLE
CLM-6: Fix view privileges issue for Result Form.

### PR DESCRIPTION
**Pull request by SHUJAAT-DEV**
Resolved the view previleges. The issue was due to incorrect privileges added in result form.
Jira ticket [link](https://issues.openmrs.org/browse/CLM-6)

### CODE REVIEW

File: omod/src/main/webapp/addLabTestResult.jsp [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <%@ include file="/WEB-INF/template/include.jsp"%>
 <%@ include file="/WEB-INF/template/header.jsp"%>
-<openmrs:require privilege="View labTestResult" otherwise="/login.htm"
+<openmrs:require privilege="View CommonLabTest results" otherwise="/login.htm"
```
Comment (seekme94) [inline omod/src/main/webapp/addLabTestResult.jsp]: Should be 'Results' not 'results'.

Comment (seekme94) [inline omod/src/main/webapp/addLabTestResult.jsp]: Unit test not included.

**Comment by dkayiwa** [comment]
@SHUJAAT-DEV  did you see the above comments? Or are you still working on this?

**Comment by SHUJAAT-DEV** [comment]
@dkayiwa yes, I'm working on these changes.

**Pull request merged**